injector/src/test/java/edu/ucr/cs/riple/injector/ClassSearchTest.java

```diff
@@ -493,14 +503,32 @@ public void proper_report_for_class_not_found_test() {
                 "javax.annotation.Nullable",
                 true))
         .start();
-    String errOutput = err.toString();
+    assert err.toString().equals(expectedErrorMessage);
```

Comment:
Don't use java assert directly in tests, prefer JUnit's assertion APIs, such as `assertEquals(...)` or `assertTrue(...)`. That way we don't depend on whether assertions are kept by the compiler or not (subject to compiler configuration)

Reply:
e8040841a6b3d56eb5d1a8798052acabcd760a11